doc/build.rst

```diff
@@ -110,8 +110,10 @@ Building the Targets
 
       git clone --recursive https://github.com/mc2-project/mc2-xgboost.git
 
-2. Configure the enclave parameters in the file ``enclave/xgboost.conf``, including the amount of virtual memory available to the enclave.
+2. Configure the enclave parameters in ``CMakeLists.txt``, including the amount of virtual memory available to the enclave.
```

Comment:
Maybe it would be helpful to specify which parameters are relevant or where in the file the user should look to change parameters, i.e. lines 13-25

Reply:
Made some improvements, let me know what you think.